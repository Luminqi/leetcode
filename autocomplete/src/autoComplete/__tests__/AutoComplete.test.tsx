import React from 'react'
import AutoComplete from '../AutoComplete'
import renderer from 'react-test-renderer'

describe('<AutoComplete />', () => {
  it('renders without crashing', () => {
    const autocomplete = renderer.create(<AutoComplete />).toJSON()
    expect(autocomplete).toMatchSnapshot()
  })
})
