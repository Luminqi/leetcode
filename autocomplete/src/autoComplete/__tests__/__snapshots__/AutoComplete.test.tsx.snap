// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AutoComplete /> renders without crashing 1`] = `
<div
  className="autocomplete"
>
  <div
    className="autocomplete-warn"
    style={
      Object {
        "visibility": "hidden",
      }
    }
  >
    <div
      className="autocomplete-warn-content"
    >
      <span>
        You input too much charactors
      </span>
    </div>
    <div
      className="autocomplete-warn-arrow"
    />
  </div>
  <div
    className="autocomplete-input-wrapper"
  >
    <input
      className="autocomplete-input"
      onChange={[Function]}
      value=""
    />
    <div
      className="autocomplete-search"
    >
      <svg
        className="autocomplete-svg search-svg"
        viewBox="0 0 1024 1024"
      >
        <path
          d="M999.592902 991.251373L712.326619 692.501398c73.271365-73.484026 118.664812-174.787972 118.664812-286.512304 0-223.873986-182.115108-405.989094-405.989094-405.989094s-405.989094 182.115108-405.989094 405.989094 182.115108 405.989094 405.989094 405.989095c98.268694 0 188.456271-35.127723 258.750383-93.416158l287.962264 299.484622a19.294148 19.294148 0 0 0 27.336599 0.541319 19.332814 19.332814 0 0 0 0.541319-27.336599zM425.002337 773.312561c-202.549892 0-367.323466-164.773574-367.323466-367.323467s164.773574-367.323466 367.323466-367.323466 367.323466 164.773574 367.323466 367.323466-164.792907 367.323466-367.323466 367.323467z"
        />
      </svg>
    </div>
    <button
      className="autocomplete-clearbtn"
      onClick={[Function]}
      style={
        Object {
          "visibility": "hidden",
        }
      }
    >
      <svg
        className="autocomplete-svg clear-svg"
        viewBox="0 0 20 20"
      >
        <path
          d="M1,1 L19,19"
        />
        <path
          d="M19,1 L1,19"
        />
      </svg>
    </button>
    <div
      className="autocomplete-button-wrpper"
    />
  </div>
</div>
`;
